### PR TITLE
racon 1.4.13

### PR DESCRIPTION
**Pull request by SeekingMeaning**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by sjackman** [comment]
On macOS
```
/tmp/racon-20200718-16773-12tzjs3/racon-v1.4.13/vendor/spoa/src/alignment_engine.cpp:37:20: error: 'invalid_argument' is not a member of 'std'
   37 |         throw std::invalid_argument("[spoa::createAlignmentEngine] error: "
      |                    ^~~~~~~~~~~~~~~~
```
https://github.com/brewsci/homebrew-bio/pull/1105/checks?check_run_id=884266939#step:4:180

**Comment by sjackman** [comment]
On Linux
```
==> brew test --verbose brewsci/bio/unicycler
104
==> FAILED
105
Testing brewsci/bio/unicycler
106
==> /home/linuxbrew/.linuxbrew/Cellar/unicycler/0.4.7/bin/unicycler --help
107
Error: brewsci/bio/unicycler: failed
108
An exception occurred within a child process:
109
  Errno::ENOENT: No such file or directory - /home/linuxbrew/.linuxbrew/Cellar/unicycler/0.4.7/bin/unicycler
110
```
https://github.com/brewsci/homebrew-bio/pull/1105/checks?check_run_id=884266950#step:5:109

Looks like maybe Unicycler needs to be revision bumped, possibly due to `python`. That could be done in a separate PR.

**Comment by SeekingMeaning** [comment]
Vendored `spoa` is version 3.0.2, and the `invalid_argument` error is fixed in 3.1.1
https://github.com/rvaser/spoa/commit/06d58ef50ab19184bb1d905443e091310de9ce2c

File: Formula/racon.rb [hunk 1]
```diff
@@ -13,14 +13,24 @@ class Racon < Formula
...
+  # Update spoa from 3.0.2 to 3.1.1 to fix 'invalid_argument' error
+  # https://github.com/rvaser/spoa/pull/28
+  resource "spoa" do
+    url "https://github.com/rvaser/spoa.git",
+      :revision => "06d58ef50ab19184bb1d905443e091310de9ce2c"
+  end
 
   def install
+    rm_rf "vendor/spoa"
+    (buildpath/"vendor/spoa").install resource("spoa")
```
Comment (sjackman) [inline Formula/racon.rb:33]: Nice fix!

**Pull request merged**